src/test/regress/output/directory_table.source

```diff
@@ -802,23 +802,75 @@ NOTICE:  dir_table1 INSERT AFTER ROW  (seg2 127.0.1.1:7004 pid=31192)
 ERROR:  syntax error at or near "WITH"
 LINE 1: ...dir_table1 FROM STDIN 'nation5' WITH TAG 'nation' WITH TAG '...
                                                              ^
+\COPY BINARY dir_table1 FROM '@abs_srcdir@/data/nation.csv' 'nation6';
+NOTICE:  dir_table1 INSERT AFTER ROW  (seg0 127.0.1.1:7002 pid=21790)
```

Comment:
Does these seg info like ip:port, pid count when matching the result?

Reply:
Yes, seg info will be ignore in regression diffs.